common/reviews/api/api-extractor.api.md

```diff
@@ -163,6 +168,12 @@ export class ExtractorResult {
     readonly warningCount: number;
 }
 
+// @public
+export interface IApiReportConfig {
```

Comment:
It's confusing to have both `IConfigApiReport` and `IApiReportConfig` in the same API.

Suggest to rename `IApiReportConfig` -> `IExtractorConfigReport` since it is conceptually just a nested part of the ExtractorConfig.